### PR TITLE
initial wormchain mainnet information

### PR DESCRIPTION
**Pull request by conorpp**
This is the genesis and initial network configuration guardians may use to join the network.  We will update the `persistent_peers` later after more guardians have joined.

I'd like to host this in the repo for people to download easily.

### CODE REVIEW

**Review by jynnantonix — APPROVED** [review]

File: wormchain/mainnet/genesis.json [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+    },
+    "genutil": {
+      "gen_txs": [
+        {
+          "body": {
+            "messages": [
+              {
+                "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+                "description": {
+                  "moniker": "jumpcrypto",
```
Comment (evan-gray) [inline wormchain/mainnet/genesis.json:144]: Is this "jumpcrypto" indicative of the expectation that they are / were the first guardian to stand up the chain?

File: wormchain/mainnet/genesis.json [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+          ]
+        }
+      ],
+      "supply": [],
+      "denom_metadata": [
+        {
+          "description": "Wormchain's native test asset",
+          "denom_units": [
+            {
+              "denom": "utest",
```
Comment (evan-gray) [inline wormchain/mainnet/genesis.json:65]: should there be a test token?

**Review by evan-gray — APPROVED** [review]
nothing immediately jumped out at me, but left a couple comments

File: wormchain/mainnet/genesis.json (hunk 1 above)
Comment (conorpp) [reply wormchain/mainnet/genesis.json:144]: Yes we will start with one node and vote the other guardians in, allowing them to join async

File: wormchain/mainnet/genesis.json (hunk 2 above)
Comment (conorpp) [reply wormchain/mainnet/genesis.json:65]: Yes I'd like to use the test token to create the guardian accounts when guardians generate their validator addresses. I.e.:
1. guardian generates validator address.
2. we send them a test token as a quick way to create the account for their address.

**Pull request merged**